onos-legacy/Chart.lock

```diff
@@ -0,0 +1,6 @@
+dependencies:
+- name: atomix
+  repository: https://charts.atomix.io
+  version: 0.1.2
+digest: sha256:e4e0a42e89042f07fc465b0360b3ca1740d2adab085ef948a9b514fb0d68255b
+generated: "2020-04-15T12:36:52.804858-07:00"
```

Comment:
Should we commit the Chart.lock and the .tgz of the chart that is depended on?

Reply:
Are you saying we should not do this? I was following what the `onos` chart in this repo did, but I don't know what the best practice is so I can remove it if we shouldn't be checking this in.

Reply:
I'd argue we shouldn't: Three points:

- It can easily be recreated locally with a `helm dep` by the client. 
- It's the artifact output of a build process - most people would question a compiled binary being included in a commit.
- There's no CI verification that the .lock is valid and that the checksum in it matches the .tgz  file.